flowclient/flowclient/client.py

```diff
@@ -1399,3 +1399,39 @@ def event_count(
         "event_types": event_types,
         "subscriber_subset": subscriber_subset,
     }
+
+
+def nocturnal_events(
+    *,
+    start: str,
+    stop: str,
+    hours: tuple((int, int)),
+    subscriber_subset: Union[dict, None] = None,
+) -> dict:
+    """
+    Return query spec for nocturnal events
+
+    Parameters
+    ----------
+    start : str
+        ISO format date of the first day of the count, e.g. "2016-01-01"
+    stop : str
+        ISO format date of the day _after_ the final date of the count, e.g. "2016-01-08"
+    hours: tuple(int,int)
+
+    subscriber_subset : dict or None, default None
+        Subset of subscribers to include in event counts. Must be None
+        (= all subscribers) or a dictionary with the specification of a
+        subset query.
+    Returns
+    -------
+    dict
+        Dict which functions as the query specification
+    """
```

Comment:
```suggestion
    """

```
(Missing blank line.)